app/store/env/go.js

```diff
@@ -2236,13 +2236,15 @@ YUI.add('juju-env-go', function(Y) {
 
       @method getChangeSet
       @param {String} bundleYAML The bundle YAML file contents.
+      @param {String} changesToken The token identifying a bundle change set
```

Comment:
bah sorry read it backwards. This clearly states the changestoken is ignored if the yaml is provided.